webapp-django/crashstats/manage/jinja2/manage/home.html

```diff
@@ -56,15 +56,6 @@
                 API Tokens
             </a></li>
         </ul>
-
-        <h3>Configuration</h3>
-        <ul>
-            <li><a href="{{ url('manage:supersearch_fields') }}">
-                <span class="fa fa-th-list fa-5x"></span>
-                Super Search Fields
-            </a></li>
-        </ul>
-
         <h3>Tools</h3>
```

Comment:
So, the whole bug was basically that 1) there was two links called "Super Search Fields" on the admin home page and 2) the view `supersearch_fields` still existed.